src/commands/Admin/admin/botfarm.ts

```diff
@@ -0,0 +1,71 @@
+import Command from "../../../structures/Command";
+import SubCommand from "../../../structures/SubCommand";
+import CommandContext from "../../../structures/CommandContext";
+import { Guild, Message } from "discord.js";
+import BulbBotClient from "../../../structures/BulbBotClient";
+import { writeFileSync } from "fs";
+
+export default class extends SubCommand {
+	constructor(client: BulbBotClient, parent: Command) {
+		super(client, parent, {
+			name: "botfarm",
+			minArgs: 1,
+			maxArgs: 1,
+			argList: ["percent:number"],
+			usage: "<percent>",
+		});
+	}
+
+	public async run(context: CommandContext, args: string[]): Promise<void | Message> {
+		this.client.guilds.cache.map(async (guild: Guild) => await guild.members.fetch());
+
+		let botfarms: any = [];
```

Comment:
```suggestion
		const botfarms: any = [];
```
Can be declared `const` as long as there's no `botfarms = .....` anywhere (another suggestion resolves this below). Declaring `const` whenever possible is widely agreed best-practice.